src/cargo/core/compiler/timings/report.rs

```diff
@@ -803,18 +808,31 @@ static HTML_CANVAS: &str = r#"
         based on the client viewport.
     -->
     <td><input type="range" min="1" max="100" value="50" id="scale"></td>
+    <td>
+        <label>
+            <input type="radio" name="renderer" value="canvas" checked />
+            Canvas
+        </label>
+        <label>
+            <input type="radio" name="renderer" value="svg" />
+            SVG
+        </label>
+    </td>
```

Comment:
I plan to make the SVG renderer the default, since it outperformed Canvas in my testing across Firefox, Chrome, and Safari!